src/exportImage.py

```diff
@@ -67,9 +67,17 @@ def export(self, bbox, filename, url, taskid, directoryName, max_zoom):
 
                     file_size = path.getsize(full_path)
                     if (self.__config["storage_provider"] == StorageProvider.S3.value):
-                        self.upload_to_s3(filename, directoryName, output_format, full_path)
-                        self.delete_local_directory(directoryName)
-                    self.__helper.save_update(
+                        try:
+                            s3_download_url = self.upload_to_s3(filename, directoryName, output_format, full_path)
+                        except Exception as e:
+                            raise e
+                        finally:
+                            self.delete_local_directory(directoryName)
+                            self.__helper.save_update(
```

Comment:
why? if failed to load to s3 than the process was NOT completed successfully

Reply:
fixed in df899c8